.github/PULL_REQUEST_TEMPLATE/develop-template.md

```diff
@@ -0,0 +1,16 @@
+Originator(s):
+
+Summary (include the keyword ['closes', 'fixes', 'resolves'] and issue number):
+
+Describe any changes made to the namelist:
+
+List all files eliminated and why:
+
+List all files added and what they do:
+
+List all existing files that have been modified, and describe the changes: 
+(Helpful git command: `git diff --name-status development...<your_branch_name>`)
+
+List any test failures:
```

Comment:
Perhaps specify what tests we expect to be run:  `List any regression test failures on xxx/yyy` (where xxx/yyy is each computer/compiler).  I think it is just one combo right now, but perhaps that might change? 

Are there other tests we currently expect developers to run?